Client/core/CAdditionalVertexStreamManager.cpp

```diff
@@ -329,6 +329,7 @@ bool CAdditionalVertexStreamManager::UpdateAdditionalStreamContent(SCurrentState
         {
             // Validate
             CVector& Normal = NormalList[i];
+            static constexpr float FLOAT_EPSILON = 0.0001f;
             if (Normal.Normalize() < FLOAT_EPSILON)
```

Comment:
`std::numeric_limits<float>::epsilon()` isn't ideal, since it's only defined to 

> the difference between 1.0 and the next value representable by the floating-point type T

which is commonly too small for a lot of floating point calculations.